src/managers/UserManager.js

```diff
@@ -37,6 +39,10 @@ class UserManager extends BaseManager {
   resolve(user) {
     if (user instanceof GuildMember) return user.user;
     if (user instanceof Message) return user.author;
+    if (typeof user === 'string') {
+      const userId = user.match(USERS_PATTERN)[1];
+      if (userId !== null) user = userId;
```

Comment:
```suggestion
      const match = user.match(USERS_PATTERN);
      if (match !== null && match[1]) user = match[1];
```
To be consistent with `resolveID()` and check for nullability if the regex doesn't match.

Reply:
`match[0]`, there's no `match[1]` without capturing groups. I think that was missed in my suggested changes somehow.

Reply:
I fixed this in the latest commit, just need to fix lint.